docs/internals/data-structures.rst

```diff
@@ -961,18 +971,24 @@ key file, wrapped using the standard ``textwrap`` module with a header.
 The header is a single line with a MAGIC string, a space and a hexadecimal
 representation of the repository id.
 
+.. _data-compression:
+
 Compression
 -----------
 
 Borg supports the following compression methods:
```

Comment:
"(each identified by two type bytes)"

Reply:
this will change in borg 2.0:

one byte for compression type, one byte for compression level.

but no need to talk about it in 1.2 docs.

Reply:
Ah, I didn't see the first comment in this thread at first. With that, the `(two NUL bytes)` is not needed, so I'll just change that up.